server/genserver.go

```diff
@@ -601,37 +602,37 @@ type paramTemplate struct {
 
 var paramTemplateStr = `
 	{{if eq .ParamType "query" -}}
-		{{.ParamName}}Strs := r.URL.Query()["{{.ParamName}}"]
+		{{.VarName}}Strs := r.URL.Query()["{{.ParamName}}"]
```

Comment:
`.VarName` is the camel-cased version of the parameter (so has dashes removed), whereas `.ParamName` is the literal name of the parameter.